ppl/zqd/handlers_test.go

```diff
@@ -575,10 +572,12 @@ func TestPostNDJSONLogWarning(t *testing.T) {
 	assert.EqualValues(t, 134, res.BytesRead)
 }
 
+// Other attempts at malformed ZSON closer to the original are not yet flagged
+// as errors. See https://github.com/brimsec/zq/issues/2057#issuecomment-803148819
```

Comment:
Another one I just wanted to make sure folks didn't miss.

Reply:
Same thing... you can leave this as is.